src/libraries/System.Net.Sockets/tests/FunctionalTests/SendTo.cs

```diff
@@ -9,7 +13,7 @@ namespace System.Net.Sockets.Tests
 {
     public abstract class SendToBase<T> : SocketTestHelperBase<T> where T : SocketHelperBase, new()
```

Comment:
Nit: the other test classes are `ReceiveFrom<T>` and `ReceiveMessageFrom<T>` -- no `Base`. Can we change to `SendTo<T>` for consistency?